scipy/signal/signaltools.py

```diff
@@ -4413,8 +4413,9 @@ def decimate(x, q, n=None, ftype='iir', axis=-1, zero_phase=True):
             n = 8
         system = dlti(*cheby1(n, 0.05, 0.8 / q))
```

Comment:
This should also be changed to have `cheby1(..., output='zpk')` so that cheby1 does not output the (lossy-precision) `b,a` form here